source/github-helpers/api.ts

```diff
@@ -186,7 +186,7 @@ export const v4 = mem(async (
 			Authorization: `bearer ${personalToken}`,
 		},
 		method: 'POST',
-		body: JSON.stringify({query: `{${query}}`}),
+		body: JSON.stringify({query: (/^(mutation )?{/.test(query.trimStart())) ? query : `{${query}}`}),
```

Comment:
Right?
```suggestion
		body: JSON.stringify({query: query.trimStart().startsWith('mutation') ? query : `{${query}}`}),
```

Reply:
Yes, I just wanted to keep it consistent with the check on line 173:

https://github.com/gamemaker1/refined-github/blob/1cd99818a6564a5c56dd56b205a35e18195bf0be/source/github-helpers/api.ts#L173